CHANGELOG.md

```diff
@@ -38,6 +38,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### Bug Fixes
+
+* (x/auth/vesting) [#16733](https://github.com/cosmos/cosmos-sdk/pull/16733) panic on overflowing and negative EndTimes when creating a PeriodicVestingAccount
```

Comment:
not: there is already a bugfix section below, can you add it there?

Reply:
Thank you @julienrbrt, updated!